From 4443945494d4920bf2ea70008f9b24fd0d0f2fde Mon Sep 17 00:00:00 2001
From: Yonghong Song <ysong@broadcom.com>
Date: Thu, 20 Dec 2012 14:50:53 -0800
Subject: [PATCH 443/565] bcm-xlp: rixi: comment out NLM_RIXI

rixi: Comment out NLM_RIXI in arch/mips/netlogic/Kconfig since we
do not really support it.

Based on Broadcom SDK 2.3.

Signed-off-by: Yonghong Song <ysong@broadcom.com>
Signed-off-by: Yanjiang Jin <yanjiang.jin@windriver.com>
---
 arch/mips/netlogic/Kconfig | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/mips/netlogic/Kconfig b/arch/mips/netlogic/Kconfig
index f6b8bc4..e6c8dd5 100644
--- a/arch/mips/netlogic/Kconfig
+++ b/arch/mips/netlogic/Kconfig
@@ -142,10 +142,10 @@ config NLM_XLP_DEVMODE
 		mode. If selected, this option enables device mode compilation.
 		Otherwise, these drivers are compiled as host mode drivers.
 
-config NLM_RIXI
-       bool "Enable Read Inhibit/ Execute Inhibit support"
-       depends on NLM_XLP && 64BIT
-       default n
+#config NLM_RIXI
+#      bool "Enable Read Inhibit/ Execute Inhibit support"
+#      depends on NLM_XLP && 64BIT
+#      default n
 
 config NLM_NET_OPTS
        bool "Enable Network Optimization (memcpy/memset)"
-- 
1.8.4.93.g57e4c17

